components/api_version/src/lib.rs

```diff
@@ -149,6 +150,8 @@ pub struct RawValue<T: AsRef<[u8]>> {
     pub user_value: T,
     /// The unix timestamp in seconds indicating the point of time that this key will be deleted.
     pub expire_ts: Option<u64>,
+    /// Logical deletion flag in apiv2, should be None in APIV1 and APIV1ttl
+    pub is_delete: Option<bool>,
```

Comment:
I think "bool" is enough. "None" seems to be unnecessary. Always be "false" in V1 and V1TTL.

Reply:
fixed